README.md

```diff
@@ -20,34 +20,68 @@ Follow the directions [here](https://golang.org/doc/) for installation, usage
 and configuration instructions. Make sure to set the [Go PATH environment
 variable](https://go.dev/doc/install) before starting the following steps.
 
-## Download & Build
+## Downloading & Building
 
-1. Downloading the latest version:
+1. Downloading the `cloudberrydb/gpbackup` repository to your target machine:
 
-```bash
-go install github.com/cloudberrydb/gpbackup@latest
-```
+    ```bash
+    git clone https://github.com/cloudberrydb/gpbackup.git
+    ```
 
-This will place the code in `$GOPATH/pkg/mod/github.com/cloudberrydb/gpbackup`.
+2. Installing the `gpbackup` package:
 
-2. Building and installing binaries
+    ```bash
+    go install github.com/cloudberrydb/gpbackup@latest
+    ```
 
-Make the `gpbackup` directory your current working directory and run:
+    This will place the code in `$GOPATH/pkg/mod/github.com/cloudberrydb/gpbackup`.
 
-```bash
-make depend
-make build
-```
+3. Entering the directory of `cloudberrydb/gpbackup`. Then, building and installing binaries of source code:
+
+    ```bash
+    cd <your_local_gpbackup_repo>
+    make depend
+    make build
+    ```
+
+    <!-- The `build` target will put the `gpbackup` and `gprestore` binaries in
+    `$HOME/go/bin`. This will also attempt to copy `gpbackup_helper` to the
+    CloudberryDB segments (retrieving hostnames from `gp_segment_configuration`).
+    Pay attention to the output as it will indicate whether this operation was
+    successful. -->
+
+    `make build_linux` is for cross-compiling on macOS, and the target is Linux.
+
+    `make install` will scp the `gpbackup_helper` binary (used with -single-data-file flag) to all hosts.
+
+4. Checking whether the build is successful by checking whether your `$HOME/go/bin` directory contains `gpback`, `gprestore`, and `gpbackup_helper`.
+
+    ```bash
+    ls $HOME/go/bin
+    ```
+
+5. If successful, copying the `gpback`, `gprestore`, and `gpbackup_helper` files to the `/bin` folder of your CloudberryDB installation on each segment. For example:
+
+    ```bash
+    cp /home/gpadmin/go/bin/gpbackup /usr/local/cloudberry-db/bin/
+    cp /home/gpadmin/go/bin/gprestore /usr/local/cloudberry-db/bin/
+    cp /home/gpadmin/go/bin/gpbackup_helper /usr/local/cloudberry-db/bin/
+    ```
```

Comment:
After successfully building the gpbackup, you can run the commands in the $GOPATH/bin shell. There should be no need to move them to the specific directories again.